cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-settings-step/UnmanagedClustersSettingsBasic.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ChangeEvent } from 'react';
+
+import { CdsControlMessage } from '@cds/react/forms';
+import { CdsInput } from '@cds/react/input';
+import { CdsButton } from '@cds/react/button';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { INPUT_CHANGE } from '../../../../state-management/actions/Form.actions';
+
+import { StepProps } from '../../../../shared/components/wizard/Wizard';
+
+export interface FormInputs {
+    CLUSTER_NAME: string;
+}
+
+function UnmanagedClusterSettings(props: Partial<StepProps>) {
+    const { handleValueChange, currentStep, goToStep, submitForm } = props;
+
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+    } = useForm<FormInputs>();
+
+    const onSubmit: SubmitHandler<FormInputs> = (data) => {
+        if (goToStep && currentStep && submitForm) {
+            goToStep(currentStep + 1);
+            submitForm(currentStep);
+        }
+    };
+
+    const handleClusterNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+        if (handleValueChange) {
+            handleValueChange(INPUT_CHANGE, 'CLUSTER_NAME', event.target.value, currentStep, errors);
+        }
+    };
+
+    return (
+        <div className="cluster-settings-container" cds-layout="m:lg">
+            <h2>Cluster settings</h2>
```

Comment:
we could create a reusable component which contains the code from line 39 to line 59. Then the shared component could be used in the advanced setting. This could reduce duplicated code.